src/Dependencies/CodeAnalysis.Metadata/CustomDebugInfoEncoder.cs

```diff
@@ -194,7 +194,10 @@ public void AddTupleElementNames(IReadOnlyCollection<(string LocalName, int Slot
                         builder.WriteInt32(info.SlotIndex);
                         builder.WriteInt32(info.ScopeStart);
                         builder.WriteInt32(info.ScopeEnd);
-                        builder.WriteUTF8(info.LocalName);
+                        if (info.LocalName != null)
```

Comment:
I looked at the EE code that consumes this with Chuck and it does tolerate null local names.
I'll stop by to understand how to test this EE scenario.